### DISCUSSION TITLE
Cannot drag cards on project board

### DISCUSSION THREAD

**Discussion by smorford-aclu** [community/community#44595] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

**Bug**

-  Team can no longer drag issue cards on our project board

**Details**

- This includes dragging cards from one column to another
- This includes dragging cards within a column

**Timeline**

- First noticed on the morning of 1/13/22
- Still experiencing on the morning of 1/17/22

**Additional Info**

- Potentially caused by https://github.blog/changelog/2023-01-12-github-issues-january-12th-update/
- Potentially related to https://github.com/orgs/community/discussions/44217

**Comment by smorford-aclu** [comment, marked as answer]
Fixed by sorting by `manual` instead of `ascending`
<img width="326" alt="image" src="https://user-images.githubusercontent.com/116025796/212954007-15eff634-4a61-4160-afea-d664e3b77f02.png">

**Comment by gdavis** [comment]
Still seems like you should be able to drag between columns, even when using non-manual sort.

**Reply by smorford-aclu** [reply]
Agree! I think that's what [this discussion](https://github.com/orgs/community/discussions/44217) is requesting.

**Comment by Charles-Johnson** [comment]
For those wondering where the sort options are:
![스크린샷, 2023-01-25 19-23-58](https://user-images.githubusercontent.com/20388960/214539062-a3fef438-8500-466c-a8d1-86698c45fed8.png)
